vault/dendron.tutorial.rich-formatting.md

```diff
@@ -2,12 +2,12 @@
 id: 8DCgctK-RMD4EeHjC5_hI
 title: Rich Formatting
 desc: ''
-updated: 1636524684640
+updated: 1639532762689
 created: 1625573403967
 nav_order: 3
 ---
 
-Dendron supports an extended Markdown syntax, which provides a lot of options for rich formatting. Take a look at some examples in this note to see what's possible.  Have the preview pane opened (`Dendron: Show Preview`) to see how these will get rendered.
+Dendron supports an extended Markdown syntax, which provides a lot of options for rich formatting. Take a look at some examples in this note to see what's possible.  Have the preview view opened (`Dendron: Show Preview`) to see how these will get rendered.
```

Comment:
> Dendron supports an extended Markdown syntax, which provides a lot of options for rich formatting. 

This is quite vague. 

> Dendron extends regular markdown with additional syntax. This covers everything in github flavored markdown as well as additional features that are unique to Dendron.

Reply:
also, in terms of layout, a lot of people are familiar with markdown. we should divide this page into `Basics` and `Advanced` and make a note in the beginning
> If you familiar with markdown syntax, skip below to `## Advanced` to see Dendron specific extensions